views/navigation.json

```diff
@@ -0,0 +1,77 @@
+{
+    "@context": "\/api\/3\/contexts\/SystemViewTemplate",
+    "@type": "SystemViewTemplate",
+    "type": "app",
+    "config": {
+        "navigation": [
+            {
+                "title": "Resources",
+                "icon": "icon icon-resources",
+                "items": [
+                    {
+                        "title": "Attachments",
+                        "icon": "icon icon-attachments",
+                        "require": {
+                            "module": "attachments",
+                            "action": "read"
+                        },
+                        "state": {
+                            "name": "main.modules.list",
+                            "parameters": {
+                                "module": "attachments"
+                            }
+                        },
+                        "editMode": false
+                    },
+                    {
+                        "title": "Email Templates",
```

Comment:
Same here as above

Reply:
For navigation, it does not append a new entry but overwrites the existing one. So we have to keep existing entries as well. In this case 'Attachment' and 'Email Templates'